TESTING.md

```diff
@@ -0,0 +1,9 @@
+Trailblazer-Rails support rails 5.2.0 and up.
+
+To run the tests, you need to have the run the following command:
+
+```
+bundle install
+bundle exec appraisal install # This will install the different version of rails
+bundle exec appraisal rake db:create db:schema:load test # Run tests
```

Comment:
This looks good!

Do you know how to run specific test or file with specific rails version using appraisal ? Maybe we can mention that as well as it comes in handy.

Reply:
added .